### DISCUSSION TITLE
expo router in the template

### DISCUSSION THREAD

**Discussion by Pkcarreno** [obytes/react-native-template-obytes#173] [General]
I was reading about the new expo router v2, and I was wondering if this is a possible future change, and what considerations should be taken into account to make such a change in the template

**Comment by yjose** [comment]
Hey @Pkcarreno , thank you for the suggestion. 
The Expo Router looks promising and we are keeping an eye on it. However, we still haven't used it on a production app internally yet, so it's hard to integrate it without making sure it's the right option for us and for others. 
Anyway, I think it's very easy to create a new project with the template and then use Expo Router instead of just `React-Navigation`, which will only require some updates to the project structure.

if you had the chance to integrate it with the template would love to hear you feedback

**Reply by Pkcarreno** [reply]
Hey @yjose, thanks for your hard work on this template.

Since the expo version was updated to version 49 I can start exploring the migration to `expo-router`, I hope to have news soon about it.
